docs/pages/includes/enterprise/docker-images.mdx

```diff
@@ -4,6 +4,10 @@ automatically rebuilt every night.
 Nightly images point to the latest version of Teleport Enterprise from the three most recent release branches.
 They are stable, and we recommend their use to easily keep your Teleport Enterprise installation up to date.
 
+These images are hosted on Amazon ECR Public. All tags under `public.ecr.aws/gravitational/teleport-ent` [are Teleport Enterprise images](https://gallery.ecr.aws/gravitational/teleport-ent).
+Versions without a architecture postfix type are `amd64` architecute. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 
```

Comment:
What is "(public.ecr.aws/gravitational/teleport:(=teleport.major_version=))" referring to? Is this an example of an image tag that uses amd64?

Reply:
Yes, that's a default image.  All are amd64 that don't have suffixes.